gallery/src/components/NxToggle/NxToggleExample.tsx

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useState } from 'react';
+
+import { NxToggle } from '@sonatype/react-shared-components';
+
+function NxToggleExample() {
+  // this example uses the `useState` hook for succinctness, but you could also manage the state manually
+  // in a class component
+  const [isRed, setIsRed] = useState(false),
+      [isBlue, setIsBlue] = useState(false),
+      [isGreen, setIsGreen] = useState(false),
```

Comment:
I wonder if we should try to come up with an example that more clearly demonstrates the use-case difference between checkboxes and toggles (whatever that difference may be).  Using the same example for both makes them seem interchangeable, and I don't think they're supposed to be.

Reply:
> I wonder if we should try to come up with an example that more clearly demonstrates the use-case difference between checkboxes and toggles (whatever that difference may be). Using the same example for both makes them seem interchangeable, and I don't think they're supposed to be.

I think this is better: https://github.com/sonatype/sonatype-react-shared-components/pull/250/commits/ebc6e803a5c3c2bfa6280f36063c2c258eaadcf0